src/server/game/Server/Packets/ItemPackets.h

```diff
@@ -70,6 +70,17 @@ namespace WorldPackets
             uint32 Amount = 0;
         };
 
+        class ItemTimeUpdate final : public ServerPacket
+        {
+        public:
+            ItemTimeUpdate() : ServerPacket(SMSG_ITEM_TIME_UPDATE, 8 + 4) { }
+
+            WorldPacket const* Write() override;
+
```

Comment:
pls rename in:


public class ClientItemTimeUpdate
{
    public ulong ItemGuid    { get; set; }
    public uint DurationLeft { get; set; }
}